Hrushal-Nikhare.md

```diff
@@ -0,0 +1,41 @@
+---
+name: "@Hrushal-Nikhare"
+project: "Chat bot"
+---
+
+# Chat bot
+
+## Summary
+What are you going to build? What does it do?
+I am planning to bulid a Chatbot powered by a rasberry pi and a esp which answers you questions using chat-gpt and waves when ever it sees you
+
+Why are you excited to build it?
+I have seen a lot of chat bots but none of them move or wave its just there being a boring piece of plastic with AI
+
+## Plan
+
+What steps are you going to take to build it? What tools are you going to use? What will you do first, second, third, etc.?
+1. Order and receive (obviously)
+2. Configure devices on local wifi
+3. Make a esp server to handle the servos
+4. Pi program to Recognize voice and its name
+5. Requests to Chat-GPT
+6. A tidy little body for our fellow robot 🤖
+ I am going to use VSCode and a old monitor just in case with a portable wifi extender 
+
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Rasberry Pi 8GB | https://www.thingbits.in/products/raspberry-pi-4-model-b-8-gb-ram?src=raspberrypi?src=raspberrypi | $97.27  |
+| Heat Sink Case  | https://www.thingbits.in/products/aluminum-case-with-dual-fan-for-raspberry-pi-4  | $9.7 |
+| SD Card         | https://www.thingbits.in/products/sandisk-32gb-class-10-microsd-memory-card-with-noobs | $5.6 |
+| HDMI Cable      | https://www.thingbits.in/products/micro-hdmi-to-standard-hdmi-cable-for-raspberry-pi-4 | $4.5 |
+| Micro Usb to C  | https://www.thingbits.in/products/usb-micro-b-to-usb-c-adapter | $1 |
+| PiCamera        | https://robu.in/product/raspberry-pi-camera-module-3/?src=raspberrypi | $35 |
+| Oled Display    | https://www.amazon.in/A2Z-Quality-Serial-Display-Arduino/dp/B07L7WT95N/ref=sr_1_4?crid=Q6S8A10G1D11&keywords=oled+arduino&qid=1673890731&sprefix=oledardui%2Caps%2C513&sr=8-4 | $5 |
+| Motor Kit       | https://store-usa.arduino.cc/products/engineering-kit-motors-backup?selectedStore=us | $46.1 |
+| Total           |                                       | $204.17 |
```

Comment:
What about shipping costs?

Reply:
tried to get things near india from the offical website